docs/api.yaml

```
@@ -653,7 +653,7 @@ tags:
   description: |-
     This is the authentication method used by the ODK Central Frontend packaged with Central Backend. Only `User`s can authenticate this way. It consists mostly of two steps:
 
-    1. **Logging in**: presenting an Email Address and a Password for verification, after which a new `Session` is created. Associated with the Session is an expiration and a bearer token. Sessions expire 24 hours after they are created.
+    1. **Logging in**: presenting an Email Address and a Password for verification, after which a new `Session` is created. Associated with the Session is an expiration and a bearer token. If a decimal integer value is present in the environment value `ODKCENTRAL_SESSION_LIFETIME`, then that value (in seconds) is taken as the session expiry. Otherwise, a session will expire 24 hours after having been created.
```

If not using an env var, maybe:

```suggestion
    1. **Logging in**: presenting an Email Address and a Password for verification, after which a new `Session` is created. Associated with the Session is an expiration and a bearer token.  By default, sessions expire 24 hours after they are created.
```